src/Polysemy/Final/Fixpoint.hs

```diff
@@ -0,0 +1,20 @@
+module Polysemy.Final.Fixpoint
+  (
+    runFixpointFinal
+  ) where
+
+import Polysemy
+import Polysemy.Final
+import Polysemy.Fixpoint
+
+import Control.Monad.Fix
+
+-----------------------------------------------------------------------------
+-- | Run a 'Fixpoint' effect through a final 'MonadFix'
+runFixpointFinal :: (Member (Final m) r, MonadFix m)
```

Comment:
This one I like. I suspect it will diverge less than the existing fixpoint stuff :)

Reply:
State semantics aren't relevant when it comes to `mfix` right? So I can elide the **Beware**.

Reply:
I think so!

Reply:
Then that's what I'll do.